tests/fixtures/marc/marc_record_all_fields.xml

```diff
@@ -88,50 +88,35 @@
         <datafield tag="600" ind1="1" ind2="0">
             <subfield code="a">Renoir, Jean,</subfield>
             <subfield code="d">1894-1979</subfield>
-            <subfield code="0">(DLC)n  79018203</subfield>
```

Comment:
I think we might want to leave these 0 and 2 subfields in the sample fixture. That way we know when they don't show up in the tests that things are working as expected.

Reply:
We had focused on keeping the fixtures slimmed down to the essentials so I think I want to keep it that way for now but I think this is worth discussing later on